### ISSUE TITLE
All JSON responses become text responses.

### ISSUE THREAD

**Issue by naoki-tomita** [timnederhoff/wiremock-client#5]
First of all I apologize for the poor English.(This message is translated by DeepL)

From sample code.

```typescript
const sampleMapping = stubForOkResponseWithBody(
    forGetRequestMatchingUrl("/sample/path"), 
    '{"greeting": "Hello world."}'
)
```

Then curl response is like below.

```
"{\"greeting\":\"Hello world.\"}'
```

This is text response. Because jsonBody has string not object.
https://github.com/timnederhoff/wiremock-client/blob/master/lib/builders/response-definition-builder.ts#L25

But wiremock expected Object type.
This [link](http://wiremock.org/docs/stubbing/)'s `Specifying the response body` section.

So I'm hoping it will be fixed.

**Comment by naoki-tomita** [comment]
I'm not in a hurry. Because there is a workaround.

**Comment by timnederhoff** [comment]
Hi, thanks for the PR! I Merged it into master, and published it on version 1.3.0
